### ISSUE TITLE
Older Android versions

### ISSUE THREAD

**Issue by zekooooo** [lmj0011/save-on-device#3]
Hi, I love the premise of your app! Using the regular "Downloads" app on my phone requires me to unblock System in my firewall, so I've had to make a workaround in Termux. It would be really nice to have a trustworthy FOSS app to save files. Unfortunately I'm still running Android 7.0. Is there any chance of getting this to work on an older version than 9? I would be really grateful!

**Comment by lmj0011** [comment]
@zekooooo 

Do you mind laying out a specific use case for someone using Android 7? I personally only use this app for copying audio files from the Recorder app into my Downloads directory (I'm running on Android 12) so I'm unaware of any use cases outside of this particular one.

For example what app are you using where you would need to use this app to download something into the Downloads directory.

Knowing this will help me in testing, after lowering the minimum SDK requirement.

**Comment by zekooooo** [comment]
I use Firefox Klar 8 and it apparently can't save files on its own so I share them to Termux where I have a script that wget's the url.

**Comment by lmj0011** [comment]
>I use Firefox Klar 8 and it apparently can't save files on its own so I share them to Termux where I have a script that wget's the url.

Is the use case: you download something from the web using FF Klar, it goes into a restricted Downloads folder (only accessible from FF Klar) and you are wanting to copy the file to a different location?

@zekooooo

**Comment by lmj0011** [comment]
I tried replicating your issue on FF Klar 8 and on the latest version of FF Klar. A file I downloaded went into the Downloads folder (emulated Nexus 4, running Android 7), if your phone is causing this issue it's possible this app won't work either.

https://github.com/lmj0011/save-on-device/releases/tag/v0.2
